### ISSUE TITLE
DPE Inspector: Modifying most of the HDR Color Grading component's properties causes the Editor to crash

### ISSUE THREAD

**Issue by LB-KacperBieniek** [o3de/o3de#16739]
**Describe the bug**
If `ed_enableDPE` is `true`, then modifying most of the _HDR Color Grading_ component's properties (for example _Shadow Color_ or _Channel Mixing Red_) causes the Editor to crash.

The issue was first observed and recorded on Linux, but the Editor.log, error.log, and error.dmp (inside error.zip) were gathered on Windows.

Call Stack:

```
Call Stack Trace:
25) 00007FF8506CDEA7 (KERNELBASE) : UnhandledExceptionFilter
24) 00007FF84FFCC888 (InProcessClient64) : (function-name not available)
23) 00007FF84FFCC4D1 (InProcessClient64) : (function-name not available)
22) 00007FF852DB5330 (ntdll) : memset
21) 00007FF852D9C876 (ntdll) : _C_specific_handler
20) 00007FF852DB221F (ntdll) : _chkstk
19) 00007FF852D614B4 (ntdll) : RtlRaiseException
18) 00007FF852DB0D2E (ntdll) : KiUserExceptionDispatcher
17) C:\o3de\Code\Framework\AzQtComponents\AzQtComponents\Components\Widgets\Slider.cpp (359) : AzQtComponents::Slider::ShowValueToolTip
16) C:\o3de-packages\packages\qt-5.15.2-rev7-windows\qt\include\QtCore\qobjectdefs_impl.h (443) : QtPrivate::QFunctorSlotObject<`AzQtComponents::SliderDouble::SliderDouble'::`2'::<lambda_1>,1,QtPrivate::List<int>,void>::impl
15) 00007FF81EB709AA (Qt5Core) : QObject::qt_static_metacall
14) 00007FF820AF2BE0 (Qt5Widgets) : QAbstractSlider::setValue
13) 00007FF820AF2AAC (Qt5Widgets) : QAbstractSlider::setSliderPosition
12) 00007FF820B89183 (Qt5Widgets) : QSlider::mousePressEvent
11) 00007FF820A478FC (Qt5Widgets) : QWidget::event
10) 00007FF820A249A1 (Qt5Widgets) : QApplicationPrivate::notify_helper
 9) 00007FF820A228C0 (Qt5Widgets) : QApplication::notify
 8) 00007FF81EB517CB (Qt5Core) : QCoreApplication::notifyInternal2
 7) 00007FF820A25BB5 (Qt5Widgets) : QApplicationPrivate::sendMouseEvent
 6) 00007FF820A70662 (Qt5Widgets) : QSizePolicy::QSizePolicy
 5) 00007FF820A6E5FE (Qt5Widgets) : QSizePolicy::QSizePolicy
 4) 00007FF820A249A1 (Qt5Widgets) : QApplicationPrivate::notify_helper
 3) 00007FF820A23A23 (Qt5Widgets) : QApplication::notify
 2) 00007FF81EB517CB (Qt5Core) : QCoreApplication::notifyInternal2
 1) 00007FF81F654915 (Qt5Gui) : QGuiApplicationPrivate::processMouseEvent
```
It should be noted that this does not occur if `ed_enableDPE` is set to `false`.

The issue occurs on both Windows and Linux.

**Steps to reproduce**
1. Execute `ed_enable 1` command in the Console.
2. Create an entity and add HDR Color Grading and PostFX Layer components to it.
3. Modify different properties on the HDR Color Grading component.

**Expected behavior**
Editor remains stable.

**Actual behavior**
Editor crashes.

**Screenshots/Video**

https://github.com/o3de/o3de/assets/87059746/6eb88434-dbbc-4987-8176-00591363dd47

[Editor.log](https://github.com/o3de/o3de/files/12646383/Editor.log)
[error.zip](https://github.com/o3de/o3de/files/12646384/error.zip) - contains error.dmp
[error.log](https://github.com/o3de/o3de/files/12646386/error.log)

**Found in Branch**
Stabilization/2310

**Commit ID from [o3de/o3de](https://github.com/o3de/o3de) Repository**
Stabilization/2310: [13f3cbe](https://github.com/o3de/o3de/commit/13f3cbe9e6acd76e5101c0c9d845c3baf986c4b5)

**Desktop/Device:**
 - Device: PC
 - OS: Linux
 - Version: Ubuntu 22.04.2 LTS
 - CPU: Ryzen 5600H
 - GPU: NVidia RTX 3070
 - Memory: 16GB

**Comment by LB-ArturZieba** [comment]
This issue was also observed on the *Vegetation Rotation Modifier* on the Stabilization/2310 (13f3cbe) branch.

**Comment by cgalvan** [comment]
This has the same root cause as https://github.com/o3de/o3de/issues/16722

**Comment by LB-KacperBieniek** [comment]
Issue verified as fixed on Stabilization/2310 ([d380ffa](https://github.com/o3de/o3de/commit/d380ffaa3bdd7b4dad07f882acc856b64b209444)) branch.
